src/components/eRACommons.js

```diff
@@ -56,9 +54,8 @@ export const eRACommons = hh(class eRACommons extends React.Component {
     }
   };
 
-  getResearcherProperties = async () => {
-    const currentUserId = Storage.getCurrentUser().dacUserId;
-    Researcher.getPropertiesByResearcherId(currentUserId).then(response => {
+  getUserInfo = async () => {
+    User.getMe().then(response => {
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/DataBiosphere/duos-ui/pullRequest?prid=6975326)
```suggestion
    User.getMe().then((response) => {
```